src/routes.js

```diff
@@ -88,10 +88,10 @@ module.exports = [
   },
   {
     method: 'GET',
-    path: '/lib/{file*}',
+    path: '/node_modules/{file*}',
     handler: {
       directory: {
-        path: 'lib'
+        path: 'node_modules'
```

Comment:
This has two advantages.
  1. Less coupling. If your files move around on the server, you do not need to change the urls in the client. Ie the client should not need knowledge of the file structure on the server to get resources, it is the job of your server to make these available in a coherent way.
  2. You do not need to expose your whole node_modules folder (poss 10s of thousands of files)